core/frontend/src/utils/version_chooser.ts

```diff
@@ -153,12 +153,19 @@ async function loadCurrentVersion(): Promise<Version> {
   }).then((response) => response.data as Version)
 }
 
-async function loadBootstrapCurrentVersion(): Promise<string> {
+async function loadBootstrapCurrentVersion(): Promise<string | undefined> {
   return back_axios({
     method: 'get',
     url: `${API_URL}/bootstrap/current/`,
     // eslint-disable-next-line no-extra-parens
   }).then((response) => (typeof response.data === 'object' ? undefined : response.data))
+    // The update process may fail and the user may be in bootstrap-backup
+    // This allows us to have the frontend working without crashing
+    // But still visible on the back
+    .catch((message) => {
+      console.warn(`Failed to fatch bootstrap version: ${message}`)
```

Comment:
I think this is worth of an alert/notification mentioning that their bootstrap might be bad.

Reply:
Done.